core/nginx-ingress/repository.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: repository-nginx-ingress
+  namespace: config-cluster-flux
+spec:
+  interval: 10m0s
```

Comment:
what does the 10 minutes interview mean here exactly

Reply:
it is to fetch the changes from nginx helm,  I have defaulted to 10mins. I think we need to refine all the intervals.

Reply:
Change to 5m and we'll review later. any worries with that?

Reply:
This is to monitor ingress helm chart repro, I think we don't to monitor very often becasue we only looking for chat versions. I was thiking to chage to 24hs.

Reply:
@ppotturi  @DaRoszko  let's change this to a longer duration e.g. 24hrs